### PR TITLE
fix: service tags support

### PR DESCRIPTION
**Pull request by rramalinga**
## Types of changes

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes if appropriate.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-effective-nsg-rt",
    "component": "prismacloud",
    "pr-id": "85",
    "commit-sha": "31778968d5c8b89000f29d36f3dc0ee972136d2d",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "backend",
    "pr-id": "1677",
    "commit-sha": "e745f97423a42f0ca468705d9a403faa148b8142",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "gaia",
    "pr-id": "799",
    "commit-sha": "bf98eafb980ec33f313a52df3cfc56ee7a2296b7",
    "pipeline": "master"
  }
]
```

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-effective-nsg-rt",
    "component": "backend",
    "pr-id": "1677",
    "commit-sha": "85afa701b7907b00146e7c804f4d1aa0ab01bebe",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "gaia",
    "pr-id": "799",
    "commit-sha": "bf98eafb980ec33f313a52df3cfc56ee7a2296b7",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "prismacloud",
    "pr-id": "85",
    "commit-sha": "31778968d5c8b89000f29d36f3dc0ee972136d2d",
    "pipeline": "master"
  }
]
```

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-effective-nsg-rt",
    "component": "backend",
    "pr-id": "1677",
    "commit-sha": "620dddaa7bd2e8f90a80894b4e9c235caba3c19f",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "gaia",
    "pr-id": "799",
    "commit-sha": "bf98eafb980ec33f313a52df3cfc56ee7a2296b7",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "prismacloud",
    "pr-id": "85",
    "commit-sha": "31778968d5c8b89000f29d36f3dc0ee972136d2d",
    "pipeline": "master"
  }
]
```

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-effective-nsg-rt",
    "component": "backend",
    "pr-id": "1677",
    "commit-sha": "826fd17ebb117923b3b87b82eaf24f61133b4f52",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "gaia",
    "pr-id": "799",
    "commit-sha": "bf98eafb980ec33f313a52df3cfc56ee7a2296b7",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "prismacloud",
    "pr-id": "85",
    "commit-sha": "16d999d9f07161250b1822ca3b76b51f14762417",
    "pipeline": "master"
  }
]
```

File: specs/+cloudnetworkrule.spec [hunk 1]
```diff
@@ -45,8 +54,6 @@ attributes:
     stored: true
     read_only: true
     omit_empty: true
-    validations:
```
Comment (abhijitherekar) [inline specs/+cloudnetworkrule.spec:48]: why are getting rid of this?

**Comment by rramalinga** [comment]
/build - automatically fired by gogo with following PRs and commit SHAs v1.0.0
```
[
  {
    "project": "fix-effective-nsg-rt",
    "component": "backend",
    "pr-id": "1677",
    "commit-sha": "2e9112041bcc9dc0292dba85d84ac958f85dfffc",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "gaia",
    "pr-id": "799",
    "commit-sha": "2cf55797fbbaf8d4ae11d558d8a4d7745c6849be",
    "pipeline": "master"
  },
  {
    "project": "fix-effective-nsg-rt",
    "component": "prismacloud",
    "pr-id": "85",
    "commit-sha": "16d999d9f07161250b1822ca3b76b51f14762417",
    "pipeline": "master"
  }
]
```

**Review by abhijitherekar — APPROVED** [review]

**Pull request merged**